### PR TITLE
Add elasticache to resolvers

### PR DESCRIPTION
**Pull request by rmbl**
Also adds the possibility to access arrays with an adjusted variableSyntax (as described in the updated readme).

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -141,15 +158,19 @@ async function getRDSValue(key, awsParameters) {
...
   ess: getESSValue,
   kinesis: getKinesisValue,
   dynamodb: getDynamoDbValue,
   rds: getRDSValue,
   ec2: getEC2Value
 }
 
-const DEFAULT_AWS_PATTERN = /^aws:\w+:[\w-.]+:[\w.]+$/
-const SUB_SERVICE_AWS_PATTERN = /^aws:\w+:\w+:[\w-.]+:[\w.]+$/
+/* eslint-disable no-useless-escape */
```
Comment (jacob-meacham) [inline src/index.js]: What is the useless escape for?

Comment (rmbl) [reply src/index.js]: eslint thinks that the escaped [] is useless. Without the escape the regex does not compile. Seems like a false positive to me.

**Comment by rmbl** [comment]
It shouldn't. The new array syntax is only possible if the variableSyntax of serverless is changed and the rest is just a new service prefix.

**Comment by jacob-meacham** [comment]
Release as 1.3.1 - Thanks!

**Pull request merged**